python/CRUD-examples/createDBObjects.sql

```diff
@@ -0,0 +1,47 @@
+#code Sample from the tutorial at https://learncodeshare.net/2015/06/02/basic-crud-operations-using-cx_oracle/
+#The following creates and populates the tables used for the tutorial
+
+CREATE TABLE lcs_people (
+ id NUMBER GENERATED BY DEFAULT AS identity,

```

Comment:
Somewhere in the code base point out that the SQL scripts need Oracle DB 12.1 and/or show how to change them to work with 11.2